examples/find_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.16)
+project(find_package CXX)
+
+if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
+    if(NOT BUILD_SHARED_LIBS)
+        set(SFML_STATIC_LIBRARIES TRUE)
+    endif()
```

Comment:
Is this necessary? I would assume that `SFML_STATIC_LIBRARIES` is implied if `NOT BUILD_SHARED_LIBS`.

Reply:
I believe so. `SFML_STATIC_LIBRARIES` is not initialized anywhere. If you omit this in static libraries builds, CI fails because you're implicitly asking for shared libraries yet only static libraries were built.

Reply:
This is confirmed by comments here:

https://github.com/SFML/SFML/blob/0e475f3f502d6f84744b9800d6c4fa058dd64043/cmake/SFMLConfig.cmake.in#L19-L25

Reply:
https://github.com/ChrisThrasher/SFML/actions/runs/3191808515

I'm testing this in CI as we speak (type?)

Reply:
I'm fine with SFMLConfig.cmake not automatically reacting to the value of BUILD_SHARED_LIBS even if that would be convenient for certain use cases.